csrankings-y.csv

```diff
@@ -720,7 +720,7 @@ Yunan Chen,Univ. of California - Irvine,http://www.ics.uci.edu/~yunanc,LGHhEQ8AA
 Yunbo Rao,UESTC,http://www.raoyunbo.cn,NOSCHOLARPAGE
 Yunbo Wang,Shanghai Jiao Tong University,http://people.csail.mit.edu/yunbo,C8bGfr0AAAAJ
 Yunbo Zhang,Rochester Institute of Technology,http://www.willyunbozhang.com,jJ5svmIAAAAJ
-Yunchao Wei,University of Technology Sydney,https://weiyc.github.io,qL9Csv0AAAAJ
+Yunchao Wei,Beijing Jiaotong University,https://weiyc.github.io,qL9Csv0AAAAJ

```

Comment:
This institution is not in the database; please just remove them entirely.

Reply:
Bump @cs-learner-ai

Reply:
I have removed it.